### ISSUE TITLE
Add a way to manage synthetic focus on ListItems

### ISSUE THREAD

**Issue by michalkvasnicak** [Byteclaw/visage#50]
Basically autocomplete sends `aria-selected` property to all options. The ones with `true` value should behave as their are focused.

It doesn't mean they must use `aria-selected` prop, but is is necessary to propagate this prop to HTML so it is semantic.
